### DISCUSSION TITLE
[M-Z] (9 Settembre 2019) Esercizio 2

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#448] [Esami & Esoneri M-Z]
### Traccia

<img width="854" alt="image" src="https://github.com/user-attachments/assets/2b2fe060-c6b6-4393-a1f3-514785b4f1a4">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20190909.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20190909.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/e55e034c-647f-4783-b334-74bfd843c9f5)
